### DISCUSSION TITLE
Cannot telnet zookeeper 2181 port when i deploy busybox in the same namespace of k8s

### DISCUSSION THREAD

**Discussion by obitoquilt** [strimzi/strimzi-kafka-operator#8357] [Q&A] [answered]
I test connection to zookeeper 2181 port, when i deploy busybox in the same namespace to zookeeper, i cannot telnet zookeeper 2181.
but i find that the kafka can connect to zookeeper 2181 using telent. Is the any difference for kafka setting?
Thanks for any help.
```bash
telnet 10.0.0.11 2181
```

**Comment by scholzj** [comment, marked as answer]
There are network policies protecting the port. So I guess that might prevent you from connecting assuming your infra supports network policies. Also, keep in mind that the port is protected by mTLS, so even if telnet connects to it you will not be able to talk with ZooKeeper.

**Reply by obitoquilt** [reply]
Thanks for your reminding.
